view/perspective/app.js

```diff
@@ -556,14 +325,13 @@ if (_realtimeEventThrottleMilliseconds !== ZERO) {
  * Passes data on to Controller to pass onto renderers.
  *
  * @param {Object} values Data returned from AJAX.
- * @param {Object} params Data from queryParams.
  */
-function loadController(values, params) {
+function loadController(values) {
```

Comment:
the params are from the url. We needed them before when we handled unnamed perspectives. However a while ago we decided to not handle the unnamed use cases, so there is no use for params.On the master branch, loading the URL: https://refocus.internal.salesforce.com/perspectives?rootSubject=Salesforce gives a "cannot GET /v1/perspectives?rootSubject=Salesforce"

Do we want to handle the unnamed perspectives? if not, there's no use for params. Otherwise I'll file a bug to get the unnamed perspectives to work